### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by adriaanzon**
- Add a shorter way to download the language files (from http://stackoverflow.com/a/18324458/5341740)
- Fixed some markdown style issues
- Updated file paths to Laravel 5 location. Some places mentioned
  Laravel 4 locations, and some both Laravel 4 & 5. I think it's better
  and less noisy to just use the most recent one. Unless there is a
  good reason not to?

### CODE REVIEW

**Comment by caouecs** [comment]
Hello,
thanks, but we need to separate Laravel 4 and 5 for folders ;)

File: README.md [hunk 1]
```diff
@@ -7,24 +7,34 @@ In this repository, you can find the lang files for the framework PHP, [Laravel
...
     * `laravel4` for Laravel4 project
     * `master` for Laravel5 project
- * Copy the folders of languages that you want, in *app/lang* folder of your application Laravel
+* Copy the folders of languages that you want, in *resources/lang* folder of your Laravel application
+
+#### Via SVN
+
+Run this in your project directory:
+
+```
```
Comment (caouecs) [inline README.md]: I don't have svn...

**Comment by adriaanzon** [comment]
Ah, you mean the folders of this repo?

I updated the pull request with the locations for Laravel 4.

**Comment by caouecs** [comment]
Thanks

**Pull request merged**